### ISSUE TITLE
Alien buildings with eyes don't report humans' location

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/gameplay#13]
This one is merely an idea: what if buildings with eyes added the enemies on radar? It would help defending on the alien side, because it would avoid having to find the enemy first. It would also make sense, because every aliens are connected to the same hivemind.

If you like the idea, I can give a shot at implementing it.

**Comment by Viech** [comment]
I like this idea.

It should be noted that by our artistic guidelines, _all_ alien buildings are supposed to have eyes.

How would you envision the range and how wide should the vision cone be? Just the same values as for alien players for simplicity?

**Comment by necessarily-equal** [comment]
do the egg and the leech have eyes? They don't seem to?

The trapper looks like it's having a very large vision cone (or even two vision cones). The overmind has 31 eyes so it probaby sees all around. Then there's the acid tube, whose vision seems narrow focused, the booster, that according to the model, sees a bit better; and the spiker that sees better on the sides (two vision cones?). The hive basically sees like the booster, but pointing upwards and the barricade does the same.

So for the vision cone, it really seems like it should be done on a case by case basis. For the vision range, it's probably ok to have the same for every building. (I never noticed that there was a range limit, so it must be fairly high)

The two things that bugs me, is that
1. The barricade won't see anything in most case. I don't think that's really an issue, but well.
2. The acid tubes will report things depending on their orientation, but will fire in all cases. Do we want to add an eye behind them?
3. Ditto for the spiker, but it's much less of an issue because of how it should be placed

**Comment by Viech** [comment]
Right, the egg and leech don't have eyes, I forgot about those. Doesn't the overmind have just one large eye though?

Acid tube firing but not giving vision doesn't sound too problematic. We could make it so that it only fires for detected humans if we really wanted to.

**Comment by necessarily-equal** [comment]
The overmind has one large eye, and has 5 groups of 6 weird blue dots 
on the bottom. It looks like the eyes on the other things, so I just 
assumed these were eyes too

**Comment by Viech** [comment]
Yeah those are eyes as well, I forgot about them. Next time I comment something I'll look stuff up before, promise!

**Comment by bmorel** [comment]
I also like the idea.

**Comment by DolceTriade** [comment]
Same.
